src/Processors/QtiV2/Out/ResponseProcessingBuilders/QtiResponseProcessingBuilder.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace LearnosityQti\Processors\QtiV2\Out\ResponseProcessingBuilders;
```

Comment:
I think it might be better to rename the namespace suffix to just `ResponseProcessing`. That makes it more extensible and adaptable later.

Reply:
Done